### ISSUE TITLE
elevate when Windows Defender is active

### ISSUE THREAD

**Issue by Mr-BeardFace** [BishopFox/sliver#121]
**Describe the bug**
If Windows Defender is running on target and the "elevate" command is run, it can cause a persistent issue that stops any exe from being executed on target. 

If Windows Defender is set to remove threats automatically, or a user manually removes the UACBypass threat that occurs when elevate is ran, Defender will not allow the 'c:\\windows\\system32\\windowspowershell\\v1.0\\powershell.exe -c " + command' registry value within HKCU\Software\Classes\exefile\shell\open\command registry to execute however will not remove the registry value. Because the registry value is still there, any time an executable is ran, Defender will stop the executable.

Code in question can be found at "github.com/bishopfox/sliver/sliver/priv/s47_windows.go"

**To Reproduce**
1. Create a payload, set listener, and execute Sliver payload as usual (on Windows with updated Defender)
2. Open Windows Defender on target
3. Run "elevate" command associated with the above session
4. A Windows Defender alert will pop up associated with UACBypass. Select to remove this threat.
5. Attempt to open any exe. ie try to run cmd within search field on taskbar

**Expected behavior**
The registry values within Software\Classes\exefile\open\command should be removed whether elevate is successful or not. At this time, if Defender stops UACBypass, the registry values will not be removed, causing the issue.

**Desktop (please complete the following information):**
 - OS: Microsoft Windows 10 Pro
 - Version 10.0.18362 Build 18362

**Additional context**
To manually fix the issue, ctrl+shift+right click desktop and select "Open Powershell window here". run "reg delete HKCU\Software\Classes\exefile\open\command" to delete registry manually.

**Comment by Mr-BeardFace** [comment]
I've attempted to reproduce my initial results and the issue appears to be inconsistent. About half the time `Software\Classes\exefile\open\command` and `Software\Classes\exefile\open` are deleted after the UACBypass threat has been identified/removed by Defender and the other half they are not. I am unsure if the values/keys are being removed by Defender or removed by Sliver. I have noticed that the issue is more likely to occur the longer you allow `elevate` command to run (waiting 15+ seconds before removing UACBypass threat with Defender).

The above was done after applying the `4a6a8bb` update. I can reattempt to reproduce the issue without this update if you feel it would be beneficial.
